var/spack/repos/builtin/packages/mongo-cxx-driver/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class MongoCxxDriver(CMakePackage):
+    """C++ Driver for MongoDB"""
+
+    homepage = "http://www.mongocxx.org"
+    url      = "https://github.com/mongodb/mongo-cxx-driver/archive/r3.4.0.tar.gz"
+
+    version('3.2.0',     sha256='e26edd44cf20bd6be91907403b6d63a065ce95df4c61565770147a46716aad8c')
```

Comment:
Can you ensure that the version number and the version in the URL match? It won't break anything, but Spack has a `spack url summary` command that checks to see how often we are able to correctly parse the version number from a URL and if that version doesn't exist in the package it will think it failed.

Reply:
Sure, I just fixed the url.